src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -89,10 +94,11 @@ public function downloadAction(Request $request, $id)
 
     /**
      * @param FileVersion $fileVersion
+     * @param string $dispositionType
      *
      * @return StreamedResponse
      */
-    protected function getFileResponse($fileVersion)
+    protected function getFileResponse($fileVersion, $dispositionType = 'attachment')
```

Comment:
@thomasduenser there exists constants for this in symfony maybe this is the right way to set the dispositionType: http://symfony.com/doc/current/components/http_foundation/introduction.html#serving-files